cop.h

```diff
@@ -1094,10 +1094,11 @@ struct context {
 
 #define CXINC (cxstack_ix < cxstack_max ? ++cxstack_ix : (cxstack_ix = cxinc()))
 
-#define G_SCALAR	2
-#define G_ARRAY		3
-#define G_VOID		1
-#define G_WANT		3
+#define G_SCALAR        2
+#define G_LIST          3
+#define G_ARRAY         3      /* name prior to 5.35.1 */
```

Comment:
To avoid it accidentally sneaking back in core code, we cold do:

```c
#ifndef PERL_CORE
#define G_ARRAY G_LIST
#endif 
```

Reply:
Good idea. Added in c5b4740f0c. PTAL